Sources/Defaults+Subscripts.swift

```diff
@@ -91,6 +91,7 @@ public extension UserDefaults {
                 // swiftlint:disable:next force_cast
                 return _value as! T
             } else if let defaultValue = key.defaultValue {
+                T._defaults.save(key: key._key, value: defaultValue, userDefaults: self)
```

Comment:
can we remove the default behavior change from this PR and create another one for it? the reason is that this is a change that breaks compatibility with the current implementation, and so we would need a major version for that vs we don't have for the property wrapper fixes

Reply:
Sorry, this was suppose to be an internal change specific to our use case and not something for this PR. Didn't realize this PR was opened against our Master branch, will fix.